### PR TITLE
Update example and add receiptsRoot response item

### LINKED ISSUE #8: eth_getBlockByHash missing receiptsRoot in the result object
Transferred from https://github.com/ethereum/wiki/issues/672

### PR DESCRIPTION
**Pull request by acolytec3**
Fixes #8

### CODE REVIEW

File: json-rpc.md [hunk 1]
```diff
@@ -1065,8 +1065,8 @@ Returns information about a block by hash.
 
 ```js
 params: [
-   '0xe670ec64341771606e55d6b4ca35a1a6b75ee3d5145a99d05921026d1527331',
-   true
+   '0xdc0818cf78f21a8e70579cb46a43643f78291264dda342ae31049421c82d21ae',
+   false
```
Comment (ChrisChinchilla) [inline json-rpc.md]: @acolytec3 Why change these? Is that hash correct? And I'm not sure if the `true` was better?

File: json-rpc.md [hunk 2]
```diff
@@ -1098,31 +1098,36 @@ params: [
 ##### Example
 ```js
 // Request
-curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getBlockByHash","params":["0xe670ec64341771606e55d6b4ca35a1a6b75ee3d5145a99d05921026d1527331", true],"id":1}'
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getBlockByHash","params":["0xdc0818cf78f21a8e70579cb46a43643f78291264dda342ae31049421c82d21ae", false],"id":1}'
```
Comment (ChrisChinchilla) [inline json-rpc.md]: @acolytec3 Same here

**Comment by ChrisChinchilla** [comment]
@acolytec3 A couple of small comments, but that aside, the rest looks good.

**Comment by ChrisChinchilla** [comment]
Seems Ok as you submitted @acolytec3 👍

**Comment by acolytec3** [comment]
Cool, thanks! Will someone pay out the bounty on gitcoin?

**Comment by ChrisChinchilla** [comment]
@acolytec3 Yes, there in the US though, so give them time to wake up :)

**Comment by acolytec3** [comment]
Hah, no worries. Me too but my alarm goes off early. 😁

**Pull request merged**